git-secrets

```diff
@@ -234,8 +234,46 @@ register_aws() {
   add_config 'secrets.patterns' '[A-Z0-9]{20}'
   add_config 'secrets.patterns' "${opt_quote}${aws}(SECRET|secret|Secret)?_?(ACCESS|access|Access)?_?(KEY|key|Key)${opt_quote}${connect}${opt_quote}[A-Za-z0-9/\+=]{40}${opt_quote}"
   add_config 'secrets.patterns' "${opt_quote}${aws}(ACCOUNT|account|Account)_?(ID|id|Id)?${opt_quote}${connect}${opt_quote}[0-9]{4}\-?[0-9]{4}\-?[0-9]{4}${opt_quote}"
+  # Prevent the addition of actual AWS account IDs. Account IDs are often
+  # included in ARNs. The preceding pattern doesn't work in this case.
```

Comment:
All of the literal additions look fine, but these two seem too broad and would result in a lot of false positives. Can you remove these two patterns?